content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Install Istio with an External Control Plane
+title: Install External Control Plane Istio
```

Comment:
I think we should stick with the original title since `External Control Plane Istio` isn't really a thing.